test/api-digester/stability-concurrency-abi.test

```diff
@@ -59,6 +59,10 @@ Func AsyncSequence.prefix(while:) is now with @preconcurrency
 Func MainActor.run(resultType:body:) has generic signature change from <T where T : Swift.Sendable> to <T>
 Func MainActor.run(resultType:body:) has mangled name changing from 'static Swift.MainActor.run<A where A: Swift.Sendable>(resultType: A.Type, body: @Swift.MainActor @Sendable () throws -> A) async throws -> A' to 'static Swift.MainActor.run<A>(resultType: A.Type, body: @Swift.MainActor @Sendable () throws -> A) async throws -> A'
 Func _runAsyncMain(_:) is now with @preconcurrency
+Func _asyncLet_get(_:_:) has mangled name changing from '_Concurrency._asyncLet_get(Builtin.RawPointer, Builtin.RawPointer) async -> Builtin.RawPointer' to '_Concurrency._asyncLet_get(Builtin.RawPointer, Builtin.RawPointer) async -> ()'
+Func _asyncLet_get(_:_:) has return type change from Builtin.RawPointer to ()
+Func _asyncLet_get_throwing(_:_:) has mangled name changing from '_Concurrency._asyncLet_get_throwing(Builtin.RawPointer, Builtin.RawPointer) async throws -> Builtin.RawPointer' to '_Concurrency._asyncLet_get_throwing(Builtin.RawPointer, Builtin.RawPointer) async throws -> ()'
+Func _asyncLet_get_throwing(_:_:) has return type change from Builtin.RawPointer to ()
```

Comment:
The declarations are annotated with `_silgen_name`, so the emitted symbols ought to remain the same even though the signature changed. Is this just a bug in the API digester that it flags it as an ABI change?

Reply:
It seems there was a patch to use mangled name instead of USR https://github.com/apple/swift/pull/39846 but reverted for some reason. cf. https://github.com/apple/swift/pull/39906 https://github.com/apple/swift/pull/39920#issuecomment-952159975

@nkcsgexi Could you tell me the reason if possible?

Reply:
OK. This change looks like it should be ABI-neutral as long as I'm not missing something.